app/controllers/concerns/sign_in/authentication.rb

```diff
@@ -63,5 +65,14 @@ def handle_authenticate_error(error)
       log_message_to_sentry(error.message, :error, context)
       render json: { errors: error }, status: :unauthorized
     end
+
+    def validate_request_ip
+      return if @current_user.fingerprint == request.ip
+
+      log_context = { request_ip: request.ip, fingerprint: @current_user.fingerprint }
+      Rails.logger.warn(SignIn::Errors::FingerprintMismatchError, log_context)
```

Comment:
Since this is just a log now, we can probably remove the error itself and just make it a raw log, something like:
`[SignIn][Authentication] fingerprint mismatch`